static/app/components/sidebar/sidebarPanel.tsx

```diff
@@ -61,7 +64,7 @@ const getSidebarPortal = () => {
   return portal as HTMLDivElement;
 };
 
-function SidebarPanel({
+export default function SidebarPanel({
```

Comment:
general observation: the mix between default exports and named exports for components seems a bit arbitrary at the moment.

default exports work better with dynamic imports, but do you think there is a world we we could enforce named exports?

Reply:
I don't see a strong case for using default exports, we barely use dynamic imports and even then, the convenience still comes at a cost. My main gripe with default exports is that with time, we rename components and forget to update the import statements, leaving lingering naming around the codebase

Reply:
In that same sense, I would like for us to stop using the separate exports statements at end of file and keep the exports more tightly coupled to the exported definition.

Reply:
I’m also team “default exports were a mistake”. Let’s see if we can discuss this (again) in the TSC 😅